### PR TITLE
Add support for i18n

### PR DESCRIPTION
**Pull request by themarcus125**

### CODE REVIEW

**Comment by amati** [comment]
OMG,...
You need to update a source file in `src` directory, not build file.
Since root's `gatsby-node.js` and `fetch.js` are built results, the modifications will disappear if we run `npm run build`.

After executing `npm run watch`, can you modify files in `src` directory and give it PR again?

**Comment by themarcus125** [comment]
> OMG,...
> You need to update a source file in `src` directory, not build file.
> Since root's `gatsby-node.js` and `fetch.js` are built results, the modifications will disappear if we run `npm run build`.
> 
> After executing `npm run watch`, can you modify files in `src` directory and give it PR again?

OH shoot, I didn't know about it T_T 
Let me update it~

**Comment by themarcus125** [comment]
> After executing `npm run watch`, can you modify files in `src` directory and give it PR again?

I just updated it by now. Could you please give it a look?

Thank you and have a nice evening~

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "strapi"
   ],
   "dependencies": {
-    "axios": "^0.18.0",
+    "axios": "0.21.1",
```
Comment (amati) [inline package.json:41]: Why is the axios version updated? (it's just question)

**Review by amati — APPROVED** [review]
LGTM! please merge it!

File: package.json (hunk 1 above)
Comment (themarcus125) [reply package.json:41]: There were vulnerabilities found in some of axios 0.18.0 dependencies... So I think it's good if we could proceed with using the newer version of aixos!!

**Pull request closed without merging**